priv/static/icon-svg/icon-logan-express.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8.5 8.5">
```

Comment:
I personally prefer the approach implied by adding this icon (the approach where we use this icon and just render it in different colors depending on the route), but it looks like the actual implementation isn't doing it that way.

Based on that, do we still need to include this icon?

Reply:
😅 good catch, we do not need to include it in that case